frontend/src/components/EOI/ExpressionOfInterestReadOnlyView.vue

```diff
@@ -1,221 +1,355 @@
+                }}
+              </p>
+            </v-col>
+          </v-row>
+          <v-label>Other</v-label>
+          <v-row>
+            <v-col cols="12" sm="12" md="4" xs="12">
+              <div
+                v-for="document in otherDocuments"
+                key="document.documentid"
+                v-if="otherDocuments"
+              >
+                <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="20">
+                  mdi-file-document-check-outline
+                </v-icon>
+                {{ document.iosas_file_name }}
+              </div>
+              <div v-else>{{ NULL_STRING }}</div>
+            </v-col>
+          </v-row>
+        </div>
+
+        <br />
+        <v-divider></v-divider>
+        <v-row>
+          <v-col cols="12" sm="12" md="12" xs="12">
+            <v-label>Additional Notes</v-label>
+            <p>{{ eoi.iosas_notes || NULL_STRING }}</p>
+          </v-col>
+        </v-row>
+      </div>
+    </div>
+  </v-container>
 </template>
 
 <script>
+import EOIFormHeader from './EOIFormHeader.vue';
 import { formatBooleanToYesNoString } from '../../utils/format';
 import { NULL_STRING } from '../../utils/constants';
 export default {
   name: 'ExpressionOfInterestReadOnlyView',
-  components: {},
+  components: {
+    EOIFormHeader,
+  },
   props: {
     eoi: {
       type: Object,
       required: true,
     },
+    draftStatusCode: {
+      type: Number,
+      required: true,
+    },
   },
   mounted() {},
   data() {
     return {
+      incorporationDocument: null,
+      certificateOfGoodStandingDocument: null,
+      otherDocuments: null,
       NULL_STRING,
     };
   },
+  created() {
+    if (this.eoi?.documents?.length > 0) {
+      this.incorporationDocument = this.eoi.documents.find(
+        ({ iosas_eoidocumenttype }) => iosas_eoidocumenttype === 100000000
+      );
+      this.certificateOfGoodStandingDocument = this.eoi.documents.find(
+        ({ iosas_eoidocumenttype }) => iosas_eoidocumenttype === 100000001
+      );
+      this.otherDocuments = this.eoi.documents.filter(
+        ({ iosas_eoidocumenttype }) => iosas_eoidocumenttype === 100000002
+      );
+    }
+  },
   methods: {
     formatBooleanToYesNoString,
+    getSchoolAuthorityName() {
+      return this.eoi?.iosas_existingauthority
+        ? this.eoi[
+            '_iosas_edu_schoolauthority_value@OData.Community.Display.V1.FormattedValue'
+          ]
+        : this.eoi.iosas_schoolauthorityname;
+    },
     getAuthorityHeadName() {
-      return this.eoi.iosas_authorityheadfirstname
+      return this.eoi?.iosas_authorityheadfirstname
         ? this.eoi.iosas_authorityheadfirstname +
             ' ' +
-            this.eoi.iosas_authorityheadname
+            this.eoi.iosas_schoolauthorityheadname
         : null;
     },
     getDesignatedHeadName() {
-      return this.eoi.iosas_designatedcontactfirstname
+      return this.eoi?.iosas_designatedcontactfirstname
```

Comment:
is it a js ternary operator? Then what is the return value for the false path?

Reply:
no, it's a null check for the eoi object, if this.eoi is null or undefined, then we can't access the field name.